<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>master_bedroom</class>
 <widget class="QWidget" name="master_bedroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
		background-color: #696969
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>771</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="main_light">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>主灯</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/主灯.png</normaloff>:/icons/主灯.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="bed_light">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>床头灯</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/床头灯.png</normaloff>:/icons/床头灯.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="dec_light">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>装饰灯</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/装饰灯.png</normaloff>:/icons/装饰灯.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="fans">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>排气扇</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/排气扇.png</normaloff>:/icons/排气扇.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="curtain">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>窗帘</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/窗帘.png</normaloff>:/icons/窗帘.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="read">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>阅读</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/阅读.png</normaloff>:/icons/阅读.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="back_home">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>回房</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/回房.png</normaloff>:/icons/回房.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="leave_home">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
   color: white;
	background-color: #808080;
}
</string>
      </property>
      <property name="text">
       <string>离房</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/离房.png</normaloff>:/icons/离房.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
}</string>
   </property>
   <property name="text">
    <string>2025-03-08 19:22:01</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="position_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
}</string>
   </property>
   <property name="text">
    <string>         主卧</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>position_label</zorder>
  <zorder>time_label</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
